### PR TITLE
feat(checkbox): add checkbox to indigo theme

### PR DESCRIPTION
**Pull request by didimmova**
### Additional information (check all that apply):
 - [ ] Bug fix
 - [x] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [ ] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_checkbox.scss [hunk 1]
```diff
@@ -30,3 +30,26 @@ $_dark-fluent-checkbox: extend($_fluent-checkbox, (
 /// @requires extend
 /// @requires $_bootstrap-checkbox
 $_dark-bootstrap-checkbox: extend($_bootstrap-checkbox);
+
+/// Generates a dark indigo checkbox schema.
+/// @type {Map}
+/// @prop {Color} tick-color [#fff] - The checked mark color.
```
Comment (SisIvanova) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_checkbox.scss:36]:
```suggestion
/// @prop {Color} fill-color [#fff] - The checked mark color.
```

Comment (SisIvanova) [reply projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_checkbox.scss:36]: You should also add documentation for the rest of the properties you are changing.

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_index.scss [hunk 2]
```diff
@@ -483,7 +483,7 @@ $dark-indigo-schema: (
     igx-card: $_dark-indigo-card,
     igx-carousel: $_dark-bootstrap-carousel,
     category-chart: $_dark-bootstrap-category-chart,
-    igx-checkbox: $_dark-bootstrap-checkbox,
+    igx-checkbox: $_dark-indigo-checkbox,
```
Comment (SisIvanova) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/dark/_index.scss:486]: You should also change the property documentation above.

File: projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_checkbox.scss [hunk 3]
```diff
@@ -108,3 +108,42 @@ $_bootstrap-checkbox: extend(
...
+/// @type {Map}
+/// @prop {Color} tick-color [igx-color: 'surface'] - The checked mark color.
+/// @prop {Map} fill-color [igx-color: ('primary', 500)] - The checked border and fill colors.
+/// @prop {Map} empty-color [igx-color: ('grays', 500)] - The unchecked border color.
+/// @prop {Map} disabled-color [igx-color: ('grays', 300)] - The disabled border and fill colors.
+/// @prop {Color} disabled-color-label [igx-color: ('grays', 300)] - The disabled label color.
+/// @prop {Number} border-radius [3px] - The border radius used for the checkbox.
+///
+/// @requires {function} extend
+/// @requires {map} $_light-checkbox
```
Comment (simeonoff) [inline projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_checkbox.scss:122]: List `$_indigo-shape-checkbox` as required.

**Pull request merged**